components/pages/privacy/consent/requests/generate-consent-request/index.jsx

```diff
@@ -48,7 +50,19 @@ export default function GenerateConsentRequest() {
       requestorInboxUrl: profile.inbox,
       options,
     });
-    setConsentId(vc.id);
+    if (vc) {
+      redirectToConsentManagementUi(
+        vc,
+        `https://localhost:3000//privacy/?signedVcUrl=${encodeURIComponent(
```

Comment:
`https://localhost:3000//privacy/` do we need to remove the `//` here? 
And get rid of localhost too?

Reply:
This has been fixed in a more recent PR and I will rebase this branch later. This PR is not ready for review as there was additional work that needed to be done.